src/cli/self_update/shell.rs

```diff
@@ -256,7 +260,20 @@ impl UnixShell for Fish {
     }
 }
 
-struct Nu;
+pub(super) struct Nu;
+
+impl Nu {
+    pub(super) fn translate_cargo_home(cargo_home: &mut String) {
```

Comment:
It looks like `cargo_home_str` should probably be a method in the `UnixShell` trait with a default implementation.

Reply:
Fixed.